openedx/core/djangoapps/appsembler/sites/admin.py

```diff
@@ -87,6 +87,8 @@ class AlternativeDomainAdmin(admin.ModelAdmin):
     )
 
 
-admin.site.unregister(User)
-admin.site.register(User, TahoeUserAdmin)
-admin.site.register(AlternativeDomain, AlternativeDomainAdmin)
+if not getattr(settings, 'TAHOE_NUTMEG_TEMP_SKIP_TEST'):
+    raise ValueError('Nutmeg Code Merge 1/6 - Note 09')
+# admin.site.unregister(User)
+# admin.site.register(User, TahoeUserAdmin)
+# admin.site.register(AlternativeDomain, AlternativeDomainAdmin)
```

Comment:
Commented code is very hard to be found. Please use the following style:

```suggestion
if not getattr(settings, 'TAHOE_NUTMEG_TEMP_SKIP_TEST'):
    # TODO: Nutmeg Code Merge 1/6 - Note 0 - enable this back after the merge is completed.
    # This will ensure the code passes in tests but fails in production until it's fixed.
    admin.site.unregister(User)
    admin.site.register(User, TahoeUserAdmin)
    admin.site.register(AlternativeDomain, AlternativeDomainAdmin)
```

Reply:
Done